bib/data/defs/centos-10.yaml

```diff
@@ -0,0 +1,94 @@
+anaconda-iso:
```

Comment:
I know this is not a new thing but I will just reiterate https://github.com/osbuild/bootc-image-builder/pull/58#discussion_r1443292447

Reply:
Uh, sorry, yeah, I really need to look into your suggestion here :(